labellers.py

```diff
@@ -14,7 +17,6 @@
     ContextWindow,
 )
 
-
```

Comment:
There should be two blank lines before and after a class definition according to pep8 conventions. Just telling this incase you find it useful. Maybe you can readd this blank line?

Reply:
Sure.